.claude/skills/adapter-ops/scripts/init_embedding_adapter.py

```
@@ -0,0 +1,643 @@
+def to_icon_name(display_name: str) -> str:
+    """Convert display name to icon filename (without extension)."""
+    return display_name.replace(" ", "")
+
+
+def fetch_url(url: str, timeout: int = 10) -> bytes | None:
+    """Fetch content from URL with error handling."""
+    try:
+        headers = {
+            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36"
+        }
+        request = Request(url, headers=headers)
```

Ruff (0.14.10)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)